_build/builder/BuildIDEIntegration.py

```diff
@@ -47,26 +46,69 @@
 # with EventGhost. If not, see <http://www.gnu.org/licenses/>.
 
 """
-This file was automatically created by the BuildStaticImports.py script.
+This file was automatically created by the BuildIDEIntegration.py script.
 Don't try to edit this file yourself.
 
-This module is not directly used by EventGhost. It only exists to help
-pylint and other tools to read the sources properly, as EventGhost is using
-a lazy import pattern.
+This module is used to assist and IDE in locating the modules that are apart 
+of EventGhost. Because EventGhost uses a dynamic import system and there are 
+no "hard coded" imports of it's modules, normally an IDE would not be able to 
+track these modules. This file never gets used by the program, it is only here 
+to provide the "hard coded" imports an IDE needs to see.
+
+Follow the instructions below to create an EventGhost programming environment.
+Here is a list of a couple of the features in your IDE that will now work 
+properly.
+
+    Code Completion
+    Function/Method/Class parameters
+    Code Inspection
+    Object Definitions
+
+Instructions for PyCharm
+    Build EventGhost with the Build IDE Integration selected. 
+    File --> Settings --> Project:{YOUR PROJECT NAM} --> Project Structure
+    +Add Content Root
+    Navigate to one level lower then this file location.
+    click on OK
+    click on Sources
+    click on Apply
+    click on OK
 """
 '''
 
 FOOTER = """
 del _tmp
 
 from Classes.IrDecoder import IrDecoder
-
-def RegisterPlugin(**dummyKwArgs):
-    pass
 """
 
-class BuildStaticImports(builder.Task):
-    description = "Build StaticImports.py"
+# BOOLEAN CreateSymbolicLinkA(
+#   LPCSTR lpSymlinkFileName,
+#   LPCSTR lpTargetFileName,
+#   DWORD  dwFlags
+# );
+
+CreateSymbolicLinkW = ctypes.windll.kernel32.CreateSymbolicLinkW
+CreateSymbolicLinkW.restype = BOOL
+
+
+def symlink(source, link_name):
+    flags = 1 if os.path.isdir(source) else 0
+    lpTargetFileName = ctypes.create_unicode_buffer(source)
+    lpSymlinkFileName = ctypes.create_unicode_buffer(link_name)
+    dwFlags = DWORD(flags)
+
+    if not CreateSymbolicLinkW(lpSymlinkFileName, lpTargetFileName, dwFlags):
+        print (
+            'Build IDE Integration: '
+            'Unable to create symbolic links user MUST '
```

Comment:
Missing punctuation.

Reply:
what punctuation???

Reply:
'Unable to create symbolic links**,** user MUST '
'be in the Administrators group**.**'